pkg/api/applications/v2/activity.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 GramLabs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"net/url"
+	"time"
+
+	"github.com/thestormforge/optimize-go/pkg/api"
+)
+
+type ActivityFeed struct {
+	HomePageURL string         `json:"home_page_url,omitempty"`
+	FeedURL     string         `json:"feed_url,omitempty"`
+	NextURL     string         `json:"next_url,omitempty"`
+	Hubs        []ActivityHub  `json:"hubs,omitempty"`
```

Comment:
Cool to see this included. Would be nice to implement this as well eventually in the application service.